webextensions/sidebar/styles/base.css

```diff
@@ -385,7 +385,7 @@ tab-item tab-label {
 }
 
 /* This is for backward compatibility about custom user style rules like https://github.com/piroor/treestyletab/issues/1363 */
-tab-item .label-content {
+tab-label::part(textspan) {
```

Comment:
_textspan_ is nice name?

Reply:
The term "span" looks just to mean "a range". I think "label-text" looks more meaningful here.